src/main/java/ItemDetail.java

```diff
@@ -0,0 +1,25 @@
+public class ItemDetail {
```

Comment:
@jongalloway Do we usually have the MIT license header here?

Reply:
No